docs/Build/dbsync.md

```diff
@@ -46,7 +46,7 @@ scripts/postgresql-setup.sh --createdb
 ##### Start cardano-db-sync-tool
 ``` bash
 cd ~/git/cardano-db-sync
-PGPASSFILE=$CNODE_HOME/priv/.pgpass cardano-db-sync-extended --config $CNODE_HOME/files/config.json --genesis-file $CNODE_HOME/files/genesis.json --socket-path $CNODE_HOME/sockets/node0.socket --schema-dir schema/
+PGPASSFILE=$CNODE_HOME/priv/.pgpass cardano-db-sync-extended --config $CNODE_HOME/files/config.json --socket-path $CNODE_HOME/sockets/node0.socket --schema-dir schema/
```

Comment:
This modification will not take into consideration the associated changes to ptn0-mainnet.json as it still uses `--config $CNODE_HOME/files/config.json` when starting cardano-db-sync-extended.

Reply:
Prereqs.sh downloads relevant config (for mainnet: ptn0-mainnet.json) as config.json.

I am working on something for praos and other combinator network (hence, this PR only refers to mainnet)